sp800-63a/sec4_ial.md

```diff
@@ -109,7 +109,7 @@ At a minimum, the applicant must be verified by a process that is able to achiev
 
 #### 4.4.1.5. Presence Requirements
 
-The CSP SHOULD perform identity proofing in-person. The CSP MAY perform remote identity proofing. The CSP SHOULD offer both in-person and remote proofing.
+In-person and remote identity proofing are allowed. The CSP SHOULD offer both in-person and remote proofing.
```

Comment:
I have been trying to think of good language, and not coming up with anything. Remote vs. in-person proofing is probably not a choice that many CSPs can make anyway. Go with the current language.